### DISCUSSION TITLE
Repeat tiles in Montage - multiple pages

### DISCUSSION THREAD

**Discussion by srini8080** [ImageMagick/ImageMagick#4708] [Digital Image Processing]
IM 6/IM 7 Linux.

I have 9 tiles that should be repeated across multiple pages until the pages are full and cover all tiles by the end

[1][2][3][4] are my tiles. Each page should have 5x2 tiles.

First page
[1][2][3][4][1]
[2][3][4][1][2]

Second page
[3][4][1][2][3]
[4][1][2][3][4]

Note that the 2nd page now has covered all tiles without any holes by the time it reached the end.

I want 2 pages of the tiles in that order.

If I change the number of unique tiles to 9 and 5x2 tiles per page, then I will get 9 pages. (90 tiles will leave no holes in the pages)

Thanks

**Comment by snibgo** [comment]
```
magick montage rose: logo: wizard: ..\toes.png rose: logo: wizard: ..\toes.png ..\toes.png -duplicate 9,0--1 -tile 5x2 x.png
```

**Reply by srini8080** [reply]
So Awesome!!!

Thanks,
